cardano-cli/test/cardano-cli-golden/files/golden/babbage/transaction-view-redeemer.out

```diff
@@ -10,11 +10,12 @@ metadata: null
 mint: null
 outputs: []
 redeemers:
-- - 0
-  - 0
-  - 42
-  - - 200
-    - 100
+- input: ed7c8f68c194cc763ee65ad22ef0973e26481be058c65005fd39fb93f9c43a20#213
```

Comment:
What do you think about "script witnessed input"? Just to make things clear.

Reply:
Addressed here: https://github.com/IntersectMBO/cardano-cli/commit/4b12878879168a9500a27716dfdabfd7bd40faf5